sources/lib/api/gcp/bigquery/_table.py

```diff
@@ -416,7 +425,7 @@ def extract(self, destination, format='csv', csv_delimiter=',', csv_header=True,
     return job
 
   def load_async(self, source, mode='create', source_format='csv',
-                 csv_delimiter=',', csv_skip_header_rows=0, encoding='utf-8',
+                 csv_delimiter=',', csv_skip_leading_rows=0, encoding='utf-8',
```

Comment:
This is a breaking change from the strict definition, but probably ok for now. Lets be careful about these...